qemu/tests/cfg/device_assignment.cfg

```diff
@@ -1,13 +1,26 @@
 - device_assignment:
     only pf_assignable
     virt_test_type = qemu
+    pre_command = "modprobe vfio-pci"
```

Comment:
sr_iov_setup() will not be called for device_assignment.  I will replace it in guest-hw.cfg. Thanks.